exercises/ex_atm.py

```diff
@@ -0,0 +1,162 @@
+"""ATM Exercise"""
+
+import sys
+
+FILE_PATH_INDEX = 1  # describe placement of the argument containing file path.
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+END_ATM_SERVICE = "-1"
+
+
+def store_file_content(file_path):
+    """Read file and move atm information to dictionary.
+    The lines of the file will contain user's id, balance and password,
+    with comma's separating them.
+
+    Args:
+        file_path (str): path to atm information file.
+
+    Return:
+        dict. with user's id as keys,
+        and user's password and balance in a list as values.
+    """
+    try:
+        atm_options = {}
+        with open(file_path, 'r') as file_pointer:
+            for line in file_pointer:
+                line_list = line.rstrip().split(',')
+                atm_options.update({line_list[0]: [line_list[1],
+                                                   float(line_list[2])]})
```

Comment:
don't save the information positional (by index).
What happen if for some reason you change the order here?
you need to change every other place you use it, and if you miss 1, your code doesn't work anymore